ogc/src/main/scala/geotrellis/server/ogc/params/ParamMap.scala

```diff
@@ -73,27 +74,41 @@ case class ParamMap(params: Map[String, Seq[String]]) {
     (getParams(field) match {
       case Some(v :: Nil) if validValues.contains(v.toLowerCase) => Valid(v.toLowerCase)
       case Some(v :: Nil) => Invalid(ParamError.InvalidValue(field, v, validValues.toList))
-      case Some(vs) => Invalid(ParamError.RepeatedParam(field))
+      case Some(_) => Invalid(ParamError.RepeatedParam(field))
       case None => Invalid(ParamError.MissingParam(field))
     }).toValidatedNel
 
   def validatedVersion(default: String): ValidatedNel[ParamError, String] =
     (getParams("version") match {
       case Some(Nil) => Valid(default)
       case Some(version :: Nil) => Valid(version)
-      case Some(s) => Invalid(ParamError.RepeatedParam("version"))
+      case Some(_) => Invalid(ParamError.RepeatedParam("version"))
       case None =>
         // Can send "acceptversions" instead
         getParams("acceptversions") match {
           case Some(Nil) =>
             Valid(default)
           case Some(versions :: Nil) =>
             Valid(versions.split(",").max)
-          case Some(s) =>
+          case Some(_) =>
             Invalid(ParamError.RepeatedParam("acceptversions"))
           case None =>
             // Version string is optional, reply with highest supported version if omitted
             Valid(default)
         }
     }).toValidatedNel
-}
\ No newline at end of file
+
+  def validatedOgcTimeSequence(field: String): ValidatedNel[ParamError, List[OgcTime]] =
+    validatedOptionalParam(field).map {
+      case Some(timeString) if timeString.contains("/") => timeString.split(",").map(OgcTimeInterval.fromString).toList
+      case Some(timeString) => OgcTimePositions(timeString.split(",").toList) :: Nil
+      case None => List.empty[OgcTime]
+    }
+
+  def validatedOgcTime(field: String): ValidatedNel[ParamError, OgcTime] =
```

Comment:
Couldn't this method be simplified to just return the `head` from the result of `validatedOgcTimeSequence`? It would avoid duplication of what looks like the same logic.